src/DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,16 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('system_tasks')
+                    ->prototype('array')
+                        ->children()
+                            ->booleanNode('enabled')->defaultTrue()->end()
```

Comment:
this is a prototype which can be extended. when i add a system-task it should be enabled by default. if i want to keep the config but want to disable it you can set the enabled flag.